Egor_Saveliev_HW2.cpp

```diff
@@ -0,0 +1,216 @@
+#include <iostream>
+#include <vector>
+
+// return how many digits in number
+int numberLength(long long number) {
+  int counter{0};
```

Comment:
можно писать вот так это будет по умолчанию
```suggestion
  int counter{};
```

Reply:
agree